modules/core/src/main/java/org/apache/ignite/internal/processors/cache/CacheMetricsImpl.java

```diff
@@ -253,13 +274,13 @@ public CacheMetricsImpl(GridCacheContext<?, ?> cctx, boolean isNear) {
 
         rmCnt = mreg.longMetric("CacheRemovals", "The total number of removals from the cache.");
 
-        putTimeNanos = mreg.longMetric("PutTime",
+        putTimeTotal = mreg.longMetric("PutTimeTotal",
             "The total time of cache puts, in nanoseconds.");
 
-        getTimeNanos = mreg.longMetric("GetTime",
+        getTimeTotal = mreg.longMetric("GetTimeTotal",
             "The total time of cache gets, in nanoseconds.");
 
-        rmvTimeNanos = mreg.longMetric("RemovalTime",
+        rmvTimeTotal = mreg.longMetric("RemovalTimeTotal",
```

Comment:
nit: Removal and Remove - we should use a single word. I suggest to use "remove"

Reply:
Done.